src/components/common/PwaPopUp.jsx

```
@@ -0,0 +1,189 @@
+import { useEffect, useRef, useState } from 'react'
+import useIsMobile from '../../hooks/header/useIsMobile'
+import favicon from '../../../public/favicon-144x144.png'
+import { IoClose } from 'react-icons/io5'
+const PwaPopUp = () => {
+  const [deferredPrompt, setDeferredPrompt] = useState(null)
+  const [showInstallButton, setShowInstallButton] = useState(false)
+  const [dontShowToday, setDontShowToday] = useState(false)
+  const [dontShow, setDontShow] = useState(false)
```

ESLint

[error] 'dontShow' is assigned a value but never used.

(no-unused-vars)